frontend/views.py

```diff
@@ -16,6 +16,11 @@ def twitch(request):
     return HttpResponse(loadTemplate.render({}, request))
 
 def verify(request):
-    loadTemplate = loader.get_template('verify.html')
-    return HttpResponse(loadTemplate.render({}, request))
+    template = loader.get_template('verify.html')

```

Comment:
```suggestion
    loadTemplate = loader.get_template('verify.html')

```
update variable name to be in line with rest of file